CMakeLists.txt

```diff
@@ -18,13 +18,15 @@ else()
     set(CMAKE_DEBUG_POSTFIX _debug)
 endif()
 
-find_package(QT NAMES Qt6 Qt5 COMPONENTS Gui REQUIRED)
-find_package(Qt${QT_VERSION_MAJOR} COMPONENTS Gui REQUIRED)
+find_package(QT NAMES Qt6 Qt5 COMPONENTS Gui Quick REQUIRED)
```

Comment:
This will pull in the Qt Quick dependency unconditionally, people who don't have the Qt Quick module may not be able to compile.
You can use `find_package(Qt${QT_VERSION_MAJOR} COMPONENTS Quick)` in a separate line to make it an optional dependency. And use `if(TARGET Qt${QT_VERSION_MAJOR}::Quick)` to judge whether this module is found or not.

Reply:
Fixed in c287322